src/showcase/app/business/business-examples/accordion-examples/basic-accordion-example/basic-accordion-example.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { AccordionDirective } from '@sbb-esta/angular-business/accordion';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'sbb-basic-accordion-example',
+  templateUrl: './basic-accordion-example.component.html',
+  styleUrls: ['./basic-accordion-example.component.scss'],
+})
+export class BasicAccordionExampleComponent implements OnInit, OnDestroy {
+  @ViewChild(AccordionDirective, { static: true }) firstAccordion: AccordionDirective;
+
+  panelOpenState = false;
+  step = 0;
+  disabled = false;
+  panelMode = 'panel 1';
+  panels = ['panel 1', 'panel 2', 'panel 3', 'panel 4', 'panel 5'];
+  accordionForm: FormGroup;
+  onRadioChange: Subscription;
+  onModeChange: Subscription;
+  onDisabledChange: Subscription;
+  onMultiChange: Subscription;
+  onHideToggle: Subscription;
+  multi = false;
+  hideToggle = false;
+
+  radioOptions = [
+    {
+      name: 'Open all',
+      value: 'openAll',
+    },
+    {
+      name: 'Close all',
+      value: 'closeAll',
+    },
+  ];
+
+  constructor() {
+    this.accordionForm = new FormGroup({
+      radioModes: new FormControl({ value: null, disabled: true }),
+      disabled: new FormControl(this.disabled),
+      multi: new FormControl(this.multi),
+      panelMode: new FormControl(this.panelMode),
+      showToggle: new FormControl(this.hideToggle),
```

Comment:
```suggestion
      hideToggle: new FormControl(this.hideToggle),
```
otherwise there are console errors